entrypoint.sh

```diff
@@ -42,7 +40,14 @@ fi
 
 ### GEOUPDATEIP
 echo "LAUNCH GEOIP UPDATE"
-/usr/bin/geoipupdate --verbose --output --database-directory="${GEOIPUPDATE_DB_DIR}"
+if [ "${GEOIPUPDATE_DRYRUN-default}" == "default" ]; then #if GEOIPUPDATE_DRYRUN is not set
```

Comment:
```suggestion
if [ "${GEOIPUPDATE_DRYRUN:-default}" == "default" ]; then #if GEOIPUPDATE_DRYRUN is not set
```

Reply:
- Unneeded comment (leftover?)
- typo on the shell "default" syntax operator: missing `: `

Reply:
nope, it's because of:
```
use default expansions (${1-default} will expand to default if not already set to something else; not to be confused with ${1:-default} which expands to default if $1 it not set or is empty
```
I think I did try first with :- or := but it failed with `-Eeu -o pipefail` I need to try again, as it was with other problems at that time

Reply:
Fair, but in that case, since it modifies the value of the variable "globally", WDYT about a dedicated instruction prior to the condition? Otherwise it hides the assignation inside the `if`